integrated_channels/blackboard/admin/__init__.py

```diff
@@ -75,3 +78,43 @@ def customer_oauth_authorization_url(self, obj):
             return format_html((f'<a href="{obj.oauth_authorization_url}">Authorize Link</a>'))
         else:
             return None
+
+@admin.register(BlackboardLearnerDataTransmissionAudit)
+class BlackboardLearnerDataTransmissionAuditAdmin(admin.ModelAdmin):
+    """
+    Django admin model for BlackboardLearnerDataTransmissionAudit.
+    """
+    list_display = (
+        "enterprise_course_enrollment_id",
+        "course_id",
+        "status",
+        "modified",
+    )
+
+    readonly_fields = (
+        "blackboard_user_email",
+        "progress_status",
+        "content_title",
+        "enterprise_customer_name",
+    )
+
+    list_per_page = 1000
+
+    class Meta:
+        model = BlackboardLearnerDataTransmissionAudit
+
+    def enterprise_customer_name(self, obj):
+        """
+        Returns: the name for the attached EnterpriseCustomer.
+        Args:
+            obj: The instance of BlackboardLearnerDataTransmissionAudit
+                being rendered with this admin form.
+        """
+        EnterpriseCustomer = get_enterprise_customer_model()
+        try:
+            ec = EnterpriseCustomer.objects.filter(
+                enterprise_customer_users__enterprise_enrollments=obj.enterprise_course_enrollment_id
+            ).first()
```

Comment:
Can we make this a helper method since it's used in multiple places, and its purpose is somewhat hard to grok at a glance?

Reply:
it adds clarity and if you're unfamiliar with the django ORM this can be unapproachable so I'm fine with this 👍